src/app/dataset/depublication/sortable-header/sortable-header.component.html

```diff
@@ -1,3 +1,8 @@
-<span class="{{ conf.cssClass }} {{ classes[current] }}" (click)="valueBump()"
-  ><span class="state-arrow">{{ conf.translateKey | translate }}</span>
+<span class="{{ conf.cssClass }} {{ classes[current] }}" (click)="valueBump()">
+  <ng-template #empty
```

Comment:
Maybe choose a different name than `empty`? Perhaps `selectAll` or something like that?